File: Boca/JAVA_2/Q4/src/CD.java
public class CD extends Produto {
    private int numFaixas;

    public CD (String nome, double preco, int numFaixas) {
        super(nome, preco);
        this.numFaixas = numFaixas;
    }

    public int getNumFaixas() {
        return numFaixas;
    }
    

}

File: Boca/CPP/07/CD.cpp
#include "CD.hpp"

CD::CD(int codigoBarras, const string& nome, double preco, int numeroFaixas) : Produto(codigoBarras, nome, preco), numeroFaixas(numeroFaixas) {}

void CD::print(ostream &out) const {
    Produto::print(out);
    out << " - " << numeroFaixas << " faixas";
}

ostream& operator<< (ostream &out, CD &cd) {
    cd.print(out);
    return out;
}

CD::~CD() {}